docusaurus/docs/getting-started/what-s-included/build-time-compression.md

```diff
@@ -0,0 +1,9 @@
+---
+id: build-time-compression
+title: Build time compression
+sidebar_label: Build time compression
+---
+
+The boilerplate includes [`@moxy/next-pre-compression`](https://github.com/moxystudio/next-pre-compression) which exports a Next.js plugin and Express middleware that compresses static assets at build time to reduce downstream overhead, with *gzip* and *brotli* compression.
+
+The server will then serve the files according to what's supported by the client browser.
```

Comment:
Hmm, is this still true? The boilerplate is now using Koa, but i don't know if it's supposed to have the Koa version of the middleware by the time of the release of this doc. In either case (whether it doesn't have pre-compression anymore, or if it uses the upcoming Koa middleware), this section should be reviewed.

Reply:
This file was indeed removed.